content/applications/hr/employees/new_employee.rst

```diff
@@ -7,13 +7,19 @@ When a new employee is hired, the first step is to create a new employee form. S
 employee form. Fill out the required information (underlined in bold) and any additional details,
 then click :guilabel:`Save`.
 
```

Comment:
I think it would be helpful to add a paragraph on instructing the user.

Reply:
Ok I can do that!